### PR TITLE
Fixes JBIDE-28842 - search for some jakarta annotations in addition t…

### PR DESCRIPTION
**Pull request by robstryker**
…o javax annotations

### CODE REVIEW

**Comment by robstryker** [comment]
@sbouchet  looks like build passes. You want to +1 or merge it?

**Review by sbouchet — APPROVED** [review]
LGTM. please wait for current CF to ends before merge.

**Pull request merged**
